_posts/2016-11-22-sustaining-research-projects.md

```diff
@@ -0,0 +1,56 @@
+---
+name: sustaining-research-projects.md
+layout: post
+title: "A guide to sustainability models for research software projects"
+date: 2016-11-22
+authors:
+  - name: Daniel S. Katz
+    url: https://github.com/danielskatz/sustaining-research-projects
+categories:
+  - blog
+tags:
+- community
+- sustainability
+---
+
+A research project often starts with a bright idea and an initial commitment of volunteer time, or perhaps,
+a fixed term grant. But what happens after that initial activity? How can the project continue to
+sustain itself? (We define sustainability as the capacity to endure. Software is sustainable if it
+will continue to be available in the future, on new platforms, and meeting new needs.
+[This is from slide 23 of http://www.slideshare.net/danielskatz/scientific-software-challenges-and-community-responses,
+though it may have been taken from somewhere else earlier - if you know where, let me know.])
+
+## A guide to sustainability models for research software projects
+
+Based on work from @nayafia, who studied how people fund their work on open source projects
+(https://github.com/nayafia/lemonade-stand), I wanted to understand how
+the projects themselves are sustained. @nayafia thought this was out of scope for her document
+(see https://github.com/nayafia/lemonade-stand/issues/1), so I forked her work and created
+
+  https://github.com/danielskatz/sustaining-research-projects
+
+My intent has been to list how projects support themselves over time. Each funding category
+ideally links to several real examples, and when possible, to a useful article or page instead of just a homepage.
+
+The categories are not mutually exclusive. For example, a project might have a foundation but also use
```

Comment:
I actually meant that the project might be part of a non-profit entity (foundation) that enables it to accept funds (e.g. apache projects within the Apache Foundation)